packages/website/src/components/learn-more-card/learn-more-card.tsx

```diff
@@ -13,7 +13,7 @@ interface LearnMoreCardProps {
 export const LearnMoreCard = ({ title, description, image, link }: LearnMoreCardProps) => {
   return (
     <Link href={link} className={style["item"]}>
-      {image && <FluentImg className={style["item-image"]} name={image} />}
+      {image && <FluentImg className={style["item-image"]} name={image} alt={image} />}
```

Comment:
I think for the FluentImg they are technically just decorative so I don't think we need to fill the alt. From mdn doc we should set it to empty string, so we could just dfo that in the `fluent-img.tsx`  `alt={props.alt ?? ""}`

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attributes

> Setting this attribute to an empty string (alt="") indicates that this image is not a key part of the content (it's decoration or a tracking pixel), and that non-visual browsers may omit it from [rendering](https://developer.mozilla.org/en-US/docs/Glossary/Rendering_engine). Visual browsers will also hide the broken image icon if the alt attribute is empty and the image failed to display.

This attribute is also used when copying and pasting the image to text, or saving a linked image to a bookmark.

Reply:
Do you think this will make the tooling that reported the issue happy?

Reply:
I would hope so as this is part of the official spec. If not then we can replace it with `alt="Illustration"`